### ISSUE TITLE
Project fails to generate bundle

### ISSUE THREAD

**Issue by cesards** [googlesamples/android-dynamic-features#20]
There is a project compilation error when generating a Bundle.

```
Install-time module 'url' has a minSdkVersion(1) different than the minSdkVersion(21) of its dependency 'base'.
```

**Comment by jabubake** [comment]
https://github.com/googlesamples/android-dynamic-features/blob/master/instant/url/build.gradle#L25
has the wrong version (1).

@keyboardsurfer : the `build.gradle` files under
https://github.com/googlesamples/android-dynamic-features/tree/master/instant (both `split` and `url` seem inconsistent)

**Comment by keyboardsurfer** [comment]
Fixed with 0bdf05ebbcfaad58d6258d761b49441f586574df.
